docs/reference/shallow.md

```diff
@@ -0,0 +1,65 @@
+---
+title: shallow
+description:
+nav: 209
+---
+
+# shallow
+
+`shallow` function lets you optimize re-renders.
```

Comment:
- shallow is a comparison function
- it can be used as a custom equality function to optimize re-renders.